tests/foreman/cli/test_syncplan.py

```diff
@@ -102,6 +112,37 @@ def _make_sync_plan(self, options=None):
 
         return make_sync_plan(options)
 
+    def validate_repo_content(
```

Comment:
Can you and @oshtaier talk and move this function to helpers.  @oshtaier is using a similar function for ui test.  See https://github.com/SatelliteQE/robottelo/pull/3274/